pkg/config/config.go

```diff
@@ -103,6 +106,7 @@ type Config struct {
 	Controllers                      Controllers
 	KubeAPIOptions                   KubeAPIOptions
 	GitRewriteRule                   bool
+	VulnerabilityCountLimit          string
```

Comment:
This should be a number. The error returned by SetConfigFromEnv will cause the controller to panic and that's the desired behavior for invalid values.